### ISSUE TITLE
NuGet extension ignores RestoreProjectStyle

### ISSUE THREAD

**Issue by mhutch** [mono/monodevelop#6869]
The NuGet extension ignores the `RestoreProjectStyle` property. The "restore" command is not available even when this property is set to "PackageReference":

```
<RestoreProjectStyle>PackageReference</RestoreProjectStyle>
```

See [the NuGet docs](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#using-packagereference-for-a-project-with-no-packagereferences) for more details.

This means transitive package references do not work correctly.

> VS bug [#764063](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/764063)

**Comment by mrward** [comment]
The problem is just that the Restore command is not available? Or something more than that?

**Comment by mhutch** [comment]
It doesn't run the store either iirc

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-15] Comment by **xamarinc**



Fixed in version 8.3.0.255 (master)



Pull Request #7712 merged by: Matt Ward  
 Author: mono  
 Commit: 2cf3653e4fcfe2ad29f1be5ce1dc517c2b85a9d2 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-17] Comment by **xamarinc**



Fixed in version 8.2.0.707 (release-8.2)



Pull Request #7712 merged by: Matt Ward  
 Author: mono  
 Commit: 2cf3653e4fcfe2ad29f1be5ce1dc517c2b85a9d2 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-21] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #7712 merged by: Matt Ward  
 Author: mono  
 Commit: 2cf3653e4fcfe2ad29f1be5ce1dc517c2b85a9d2 (xamarin/monodevelop)
